axelor-sale/src/main/java/com/axelor/apps/sale/service/saleorder/SaleOrderLineServiceImpl.java

```diff
@@ -300,6 +300,13 @@ public SaleOrderLine resetProductInformation(SaleOrderLine line) {
     return line;
   }
 
+  public void resetPrice(SaleOrderLine line) {
+    if (!line.getEnableFreezeFields()) {
+      line.setPrice(null);
+    }
+    line.setInTaxPrice(null);
```

Comment:
Not sure about this, if fields are frozen, then this will empty inTaxPrice and it will not be updated by fillPrice, emptying the inTaxPrice value while we want to keep the original value ?

Reply:
Yes you're right, fixed it